app/uk/gov/hmrc/transitmovementsvalidator/controllers/MessagesController.scala

```diff
@@ -31,49 +30,62 @@ import uk.gov.hmrc.transitmovementsvalidator.models.errors.BaseError
 import uk.gov.hmrc.transitmovementsvalidator.models.errors.InternalServiceError
 import uk.gov.hmrc.transitmovementsvalidator.models.errors.UnknownMessageTypeValidationError
 import uk.gov.hmrc.transitmovementsvalidator.models.response.ValidationResponse
-import uk.gov.hmrc.transitmovementsvalidator.services.ValidationService
+import uk.gov.hmrc.transitmovementsvalidator.services.XmlValidationService
+import uk.gov.hmrc.transitmovementsvalidator.services.JsonValidationService
 
 import javax.inject.Inject
 import scala.concurrent.ExecutionContext
-import scala.concurrent.Future
 import scala.util.control.NonFatal
 
-class MessagesController @Inject() (cc: ControllerComponents, validationService: ValidationService)(implicit
+class MessagesController @Inject() (cc: ControllerComponents, XmlValidationService: XmlValidationService, JsonValidationService: JsonValidationService)(implicit
```

Comment:
_really_ nitpicky, but the variable names should start with a lower case letter.